gpbft/options.go

```diff
@@ -35,7 +35,13 @@ func newOptions(o ...Option) (*options, error) {
 }
 
 // WithDelta sets the expected bound on message propagation latency. Defaults to
-// 2 seconds if unspecified. Delta must be larger than zero.
+// 3 seconds if unspecified. Delta must be larger than zero.
+//
+// The default of 3 seconds for the value of Dela is based on the current EC
+// propagation time (i.e. round-trip time of 6 seconds) as the initial
+// propagation timeout. The actual propagation time may be significantly lower.
```

Comment:
EC doesn't assume a 6s _round-trip time_, as that's not even well-defined for a pubsub network. What it assumes is an upper bound on the gossipsub network-wide propagation time, which is actually 10s now, following https://github.com/filecoin-project/lotus/pull/9290. 

The increase from 6s to 10s was due to having time to spare in the epoch envelope due to faster proof computation; it does not imply the need to go higher than 6s. The value remains user-configurable, 6s worked well, and we have reason to believe that we can go lower.

I think we can even skip talking about EC and just say that we have reason to believe, based on previous data, that 3s is enough.

Reply:
Noted thank you; merge queue got the PR merged. I have take note to update the comments.

Reply:
Oops, forgot we had it enabled in the repo 🤦

Reply:
It's my fault. I added this pr to queue before your excellent review.